### PR TITLE
Added dynamic links in assets page

### PR DESCRIPTION
**Pull request by niklabh**
**Pull Request Summary**

Added dynamic links in asset page for
- Portal decentralised version
- Astar home
- Astar docs
- Astar blog

Screenshot 2023-05-08 at 4 49 58 PM

**Check list**

- [ ] contains breaking changes
- [ ] adds new feature
- [ ] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

**Release notes:**

- feat: Added dynamic links in asset page

**This pull request makes the following changes:**

**Adds**

- (ex: Add feature A)

**Fixes**

- (ex: Fix validation function)

**Changes**

- (ex: Change document B)

**To-dos**

> \*Feel free to remove this section if it's not applicable

- [ ] (ex: add user list)

### CODE REVIEW

File: src/components/assets/DynamicLinks.vue [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  components: {},
+  setup() {
+    const banners = [
+      require('/src/assets/img/banner/banner01.svg'),
+      require('/src/assets/img/banner/banner02.svg'),
+      require('/src/assets/img/banner/banner03.svg'),
+      require('/src/assets/img/banner/banner04.svg'),
+    ];
+    const items = linksData.map((item) => ({
+      background: banners[item.background],
```
Comment (impelcrypto) [inline src/components/assets/DynamicLinks.vue:36]: maybe use index instead of `item.background`?

File: src/data/dynamic_links.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+[
+    {
+      "background": 0,
+      "title": "Go Decentralized!",
+      "subtitle": "Portal is now on IPFS!",
```
Comment (impelcrypto) [inline src/data/dynamic_links.json:5]: Let's use i18n here

File: src/components/assets/DynamicLinks.vue [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+      title: item.title,
```
Comment (impelcrypto) [inline src/components/assets/DynamicLinks.vue:37]: Let's use i18n in title and subtitle

**Review by impelcrypto — CHANGES_REQUESTED** [review]

**Comment by impelcrypto** [comment]
@niklabh It's not responsive, and it breaks other UIs. Perhaps you can refer to dApp staking page  
image

image

**Review by bobo-k2 — CHANGES_REQUESTED** [review]
The code LGTM, but please update Release notes section in the PR description

**Comment by Kahonnohak** [comment]
Thanks @impelcrypto!!!

@gluneau Could you help @niklabh add the latest blog here, please?

**Comment by niklabh** [comment]
> @niklabh It's not responsive, and it breaks other UIs. Perhaps you can refer to dApp staking page (scroll-able) image
> 
> image

Fixed

**Review by impelcrypto — APPROVED** [review]

**Comment by Kahonnohak** [comment]
Hi @niklabh 

Please check the ticket again, no need to trim the sides but only the top and bottom. And that should end up being able to scroll to the right, like dApp Staking. Could you also check that those banners would not be on the tooltip please?

Screenshot 2023-05-10 at 11 12 19

Screenshot 2023-05-10 at 11 17 37
Screenshot 2023-05-10 at 11 17 46

**Comment by niklabh** [comment]
> @niklabh Please do not merge this PR before the production page has been updated.

Link changed. Can this be merged now?

**Comment by Kahonnohak** [comment]
No @niklabh the styling has not been fixed. Please see my last comment.

Also @gluneau please check, I could not get to the blog post.
https://gyazo.com/1d3bdfe49cba959c0733bf3a57a22da3

**Comment by niklabh** [comment]
Hey @sirius651  can you please check the scrolling styling

**Comment by sirius651** [comment]
> Hey @sirius651 can you please check the scrolling styling

not sure exact output, but fixed as same as dapp staking card scrolling. please check @niklabh @Kahonnohak

**Comment by Kahonnohak** [comment]
Thank you Will! Awesome!

Could I ask you
1) mobile to have no padding/margin on the right (same as dApp Staking)
2) Fix over-wrapping ads on the tooltip
3) Now desktop ads heights are now squashed, please take a look?

(this is not included in the release so no need to spend your evening for this)

Screenshot 2023-05-11 at 14 29 10
Screenshot 2023-05-11 at 14 33 38


> > Hey @sirius651 can you please check the scrolling styling
> 
> not sure exact output, but fixed [here](https://github.com/AstarNetwork/astar-apps/pull/775/commits/19b814795e153865278f93ce5b78f688d92c36e7) as same as dapp staking card scrolling. please check @niklabh @Kahonnohak

**Comment by sirius651** [comment]
> Thank you Will! Awesome!
> 
> Could I ask you
> 
> 1. mobile to have no padding/margin on the right (same as dApp Staking)
> 2. Fix over-wrapping ads on the tooltip
> 3. Now desktop ads heights are now squashed, please take a look?
> 
> (this is not included in the release so no need to spend your evening for this)
> 

Fixed above [here](https://github.com/AstarNetwork/astar-apps/pull/775/commits/8fe65bdf94e4b92b745ca80ee1afc389e769e645) , please check again

**Comment by Kahonnohak** [comment]
Thanks @sirius651 !

Could I ask you to make "Go Decentralized" in italic and weight 800? And styling is done! (for future, would it be simple to add animations to auto-scroll?)

@niklabh would you think "Portal is now on IPFS!" makes sense? Any better wording?

**Comment by sirius651** [comment]
> 

Updated as you suggested. I didn't catch the animations to auto-scroll. any reference for this?

**Comment by Kahonnohak** [comment]
> > Thanks @sirius651 !
> > Could I ask you to make "Go Decentralized" in italic and weight 800? And styling is done! (for future, would it be simple to add animations to auto-scroll?)
> > @niklabh would you think "Portal is now on IPFS!" makes sense? Any better wording?
> 
> Updated as you suggested. I didn't catch the animations to auto-scroll. any reference for this?

Thank you! No worries for now. Styling done. Thanks for the help!

**Comment by Kahonnohak** [comment]
Created a ticket for adding another banner on top of the header for the decentralised version, would you work on this or a new PR?

**Comment by impelcrypto** [comment]
@Kahonnohak 

> Created a ticket for adding another banner on top of the header for the decentralised version, would you work on this or a new PR?

I'll create another PR for this

File: src/pages/AssetsPage.vue [hunk 4]
```diff
@@ -52,9 +52,9 @@ export default defineComponent({
 </script>
 <style lang="scss" scoped>
 #assets-top {
-  padding: 0 16px;
```
Comment (impelcrypto) [inline src/pages/AssetsPage.vue:55]: Could you advise why deleted here for mobile devices? @Kahonnohak @niklabh It makes the divider size longer.

image

Comment (impelcrypto) [reply src/pages/AssetsPage.vue:55]: @sirius651

Comment (impelcrypto) [reply src/pages/AssetsPage.vue:55]: @sirius651  Could you fix it?

Comment (impelcrypto) [reply src/pages/AssetsPage.vue:55]: @sirius651 or you can fix it in [this](https://github.com/AstarNetwork/astar-apps/pull/803) PR.

Comment (impelcrypto) [reply src/pages/AssetsPage.vue:55]: @sirius651 I fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/803/commits/486415b6caf7ecf2548f97d75bad837ac2b2ba47) commit.

**Pull request merged**